tests/components/griddy/test_config_flow.py

```diff
@@ -0,0 +1,54 @@
+"""Test the Griddy Power config flow."""
+import asyncio
+
+from asynctest import MagicMock, patch
+
+from homeassistant import config_entries, setup
+from homeassistant.components.griddy.const import DOMAIN
+
+
+async def test_form(hass):
+    """Test we get the form."""
+    await setup.async_setup_component(hass, "persistent_notification", {})
+    result = await hass.config_entries.flow.async_init(
+        DOMAIN, context={"source": config_entries.SOURCE_USER}
+    )
+    assert result["type"] == "form"
+    assert result["errors"] == {}
+
+    with patch(
+        "homeassistant.components.griddy.config_flow.AsyncGriddy.async_getnow",
+        return_value=MagicMock(),
+    ), patch(
+        "homeassistant.components.griddy.async_setup", return_value=True
+    ) as mock_setup, patch(
+        "homeassistant.components.griddy.async_setup_entry", return_value=True,
+    ) as mock_setup_entry:
+        result2 = await hass.config_entries.flow.async_configure(
+            result["flow_id"], {"loadzone": "LZ_HOUSTON"},
+        )
+
+    assert result2["type"] == "create_entry"
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=5156575)